integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/tpch/TpchLikeBench.scala

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
2020

Reply:
So this file already existed and I renamed it. Let me see if I can rebase so git recognizes this is a rename rather than a delete and add.

Reply:
yeah if it was a git mv then its ok, we should try to make sure its a move so we don't lose history

Reply:
I have rebased and if you look at the individual commits you can see that I did a `git mv` in the first commit and then changed the file contents in the second commit but the "files changes" tab in the github UI for this PR still shows it as a delete and add for some reason. I do see the correct history locally.

Reply:
as long as its a git move its fine.